core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/DiagnosticsV4Controller.java

```diff
@@ -0,0 +1,49 @@
+package com.sequenceiq.cloudbreak.controller.v4;
+
+import javax.inject.Inject;
+import javax.validation.Valid;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+
+import com.sequenceiq.authorization.annotation.DisableCheckPermissions;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.diagnostics.DiagnosticsV4Endpoint;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.diagnostics.model.DiagnosticsCollectionRequest;
+import com.sequenceiq.cloudbreak.core.flow2.service.DiagnosticsTriggerService;
+import com.sequenceiq.cloudbreak.service.CloudbreakRestRequestThreadLocalService;
+import com.sequenceiq.cloudbreak.telemetry.VmLogsService;
+import com.sequenceiq.cloudbreak.telemetry.converter.VmLogsToVmLogsResponseConverter;
+import com.sequenceiq.common.api.telemetry.response.VmLogsResponse;
+
+@Controller
+@DisableCheckPermissions
+public class DiagnosticsV4Controller implements DiagnosticsV4Endpoint {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DiagnosticsV4Controller.class);
+
+    @Inject
+    private CloudbreakRestRequestThreadLocalService crnService;
+
+    @Inject
+    private DiagnosticsTriggerService diagnosticsTriggerService;
+
+    @Inject
+    private VmLogsService vmLogsService;
+
+    @Inject
+    private VmLogsToVmLogsResponseConverter vmlogsConverter;
+
+    @Override
+    public void collectDiagnostics(@Valid DiagnosticsCollectionRequest request) {
```

Comment:
as discussed, pls authorize this as it is done in related controller of datalake service